.github/actions/inject-issue-id-action/action.yaml

```diff
@@ -0,0 +1,99 @@
+---
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2024 The Linux Foundation
+
+name: "🎟️ Add JIRA ticket to commit message"
+
+inputs:
+  ISSUE_ID_LOOKUP_JSON:
+    description: "JSON mapping users to JIRA tickets"
+    type: string
+    required: true
+  INJECT:
+    # When injection is enabled and no issue found...
+    # String will be added, output variable set true
+    description: "Whether to inject an issue-id if NOT found"
+    type: boolean
+    required: false
+    default: true
+
+outputs:
+  PRESENT:
+    description: "Whether the required ticket string is present"
+    value: ${{ steps.status.outputs.present }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.head_ref }}
+        fetch-depth: 0
+
+    - name: "Get GIT commit message body"
+      id: commit-message
+      # yamllint disable rule:line-length
+      uses: ModeSevenIndustrialSolutions/issue-id/.github/actions/git-commit-message-action@main
```

Comment:
This is not pointing to the micro-action that is part of the PR

Reply:
Why is it I always forget to update these self-references when submitting patches?